packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1573,6 +1578,8 @@ class EditableText extends StatefulWidget {
   ///
   ///  * [TapRegion] for how the region group is determined.
   ///  * [onTapOutside], which is called for each tap down.
+  ///  * [EditableTextTapOutsideIntent] for the intent that is invoked if
```

Comment:
Really small nit: change to active to make it easier to read. So instead:

```suggestion
  ///  * [EditableTextTapOutsideIntent], the intent that is invoked if
```